src/libpriv/rpmostree-core.c

```diff
@@ -1328,11 +1319,7 @@ rpmostree_context_prepare_install (RpmOstreeContext    *self,
       path = g_strdup_printf ("%s/%s.rpm", self->metadata_dir_path, nevra);
       pkg = dnf_sack_add_cmdline_package (sack, path);
       if (!pkg)
-        {
-          g_set_error (error, G_IO_ERROR, G_IO_ERROR_FAILED,
-                       "Failed to add local pkg %s to sack", nevra);
-          goto out;
-        }
+        glnx_throw (error, "Failed to add local pkg %s to sack", nevra);
```

Comment:
(I just spotted that while looking more carefully to see if something actually caused
`(/var/tmp/checkout/.libs/lt-rpm-ostree start-daemon:17819): rpm-ostreed-WARNING **: error loading gpg verify result GPG: Unable to complete signature verification: GPGME: No data`)